name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/preferences/PPIDPreferencePage.java

```diff
@@ -48,11 +47,9 @@ protected Control createContents(Composite parent)
         this.oauthClient.addStatusListener(updateListener);
         parent.addDisposeListener(event -> oauthClient.removeStatusListener(updateListener));
 
-        Composite area = new Composite(parent, SWT.NONE);
+        var area = new Composite(parent, SWT.NONE);
         GridLayoutFactory.swtDefaults().numColumns(2).spacing(5, 10).applyTo(area);
 
-        // If we set the description on the preference page itself, then the
-        // layout of the page is broken.
```

Comment:
I spend two hours debugging this... I need some reminder in the code why the 'description' is a custom field here instead of just using #setDescription. Otherwise I fear in a couple years I will attempt to refactor this again. Let me update the comment.  😄